plugins/diff/erofs/differ.go

```diff
@@ -0,0 +1,144 @@
+//go:build linux
```

Comment:
You can drop this line by adding `_linux.go` suffix to the file name

Reply:
> You can drop this line by adding `_linux.go` suffix to the file name

Thanks! let me revise it now.

Reply:
done, thanks!